### DISCUSSION TITLE
shen-ping/info/gvvIv_FUsOJH-dSVaFXST

### DISCUSSION THREAD

**Discussion by giscus** [wurencaideli/dumogu-ypwz-comment#12] [Announcements]
# shen-ping/info/gvvIv_FUsOJH-dSVaFXST

不是只有视频和音乐才带给人欢乐！还有一篇文字哟！学生时代所看过的短文还有谁记得呢？甚是怀念呢！

https://www.dumogu.top/shen-ping/info/gvvIv_FUsOJH-dSVaFXST

**Comment by wurencaideli** [comment]
笑死我了
